.gitlab-ci.yml

```diff
@@ -98,11 +98,11 @@ check-line-width:
 
 publish-draft-release:
   stage:                           test
-  image:                           parity/tools:latest
   only:
     - tags
     - /^v[0-9]+\.[0-9]+\.[0-9]+.*$/        # i.e. v1.0.1, v2.1.0rc1
   script:
+    - apt-get -y update; apt-get -y install jq
```

Comment:
basically i don't mind, but feel like it would be cleaner to add is simply to the ci-linux image.